core/trino-main/src/main/java/io/trino/FeaturesConfig.java

```diff
@@ -948,6 +949,18 @@ public FeaturesConfig setSkipRedundantSort(boolean value)
         return this;
     }
 
+    public boolean isComplexExpressionPushdownEnabled()
+    {
+        return complexExpressionPushdownEnabled;
+    }
+
+    @Config("optimizer.complex-expression-pushdown-enabled")
```

Comment:
`optimizer.complex-expression-pushdown.enabled`

(we favor `.enabled` notation so that it looks nice when we introduce additional subproperties)

Reply:
Changed